index.md

```diff
@@ -1,209 +0,0 @@
-# Cozy Code Guidelines
-
-## Generic code style
-
-* lines must not be longer than 80 characters
-* indentation must be done with 4 spaces
-* file names should be in snake\_case
-* variables name must not be abbreviated
-* class names should be CamelCase, starting with uppercase
-* two blank lines between functions
-* one blank line at the end of files.
-
-## CSS / Stylus

```

Comment:
Well, the new ones and old ones have a lot in common, but the old ones are not our reference anymore. So kinda deprecated, yeah.